sr_unix/ydbenv.mpt

```diff
@@ -98,7 +106,8 @@ set	; Set up environment (environment variables, database, etc.) with default st
 	set tmp2=$zsearch(tmp1_"/libyottadbutil.so",$increment(zstmp))
 	set:'$zlength(tmp2) tmp2=$zsearch(tmp1_"/libgtmutil.so",$increment(zstmp))
 	set path=path_" "_tmp2
-	write "export ydb_routines=""",path,"""",!,"export gtmroutines=""$ydb_routines""",!
+	set tmp1=$ztrnlnm("ydb_routines"),tmp2=$ztrnlnm("gtmroutines")
+	write:(($g(tmp1)="$ydb_dist/libgtmutil.so")!($g(tmp2)="$ydb_dist/libgtmutil.so")) "export ydb_routines=""",path,"""",!,"export gtmroutines=""$ydb_routines""",!
```

Comment:
Since tmp1 and tmp2 are set in the previous line, there should be no need to $get() them. Use $get() rather than $g(). Also our coding practice is to put the constant first in a comparison. So, the line should read:

    write:"$ydb_dist/libgtmutil.so"=tmp1!("$ydb_dist/libgtmutil.so=tmp2) "export ydb_routines=""",path,"""",!,"export gtmroutines=""$ydb_routines""",!

Reply:
fixed.